wordpress-mobile/WordPress-Android
# Insights management tweaks

Fixes #10974

This PR contains 3 changes. 
- It replaces the "Edit" button we used to access Insight management with "Add new stats card" button. 
- It updates the insights management screen design based on the issue
- The default insights are now changed (in the linked FluxC PR - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1460)

<img width="355" alt="Screenshot 2019-12-20 at 10 33 38" src="https://user-images.githubusercontent.com/1079756/71245333-c0409f00-2314-11ea-9d2a-b17c56e9e334.png">

![Screenshot_1576834423](https://user-images.githubusercontent.com/1079756/71245326-bd45ae80-2314-11ea-8435-718ed4e12efa.png)


To test:
- Open Stats/Insights
- Scroll down
- Check that the "Add new stats card" is visible and has the updated design
- Click on it
- Check that you see insights with the updated design
- Try to add/remove a few insight types
- Click on Save
- Check that the cards are added/removed/updated
- Click on the "Add new stats card" again
- Try to add/remove a few insight types
- Cancel the changes without saving
- The cards are not added/removed/updated

PR submission checklist:
